### PR TITLE
Allow single quotes in visual pane, hints etc.

### PR DESCRIPTION
**Pull request by jaydhulia**

### CODE REVIEW

File: app/public/templates/scene.html [hunk 1]
```diff
@@ -125,12 +125,17 @@
...
             class="link"
             animation__onclick="property: position; dur: 0; startEvents: click"
-            animation-on-click-run='blackboardData: {% raw json.dumps(object["animations_json"]["blackboardData"]) %}'
+            animation-on-click-run='blackboardData: {{ json_encode(object["animations_json"]["blackboardData"]) }}'
           {% end %}
         ></a-gltf-model>
         {% end %}
       </a-scene>
       <script>
+        function htmlDecode(input) {
```
Comment (ahmedhamodi) [inline app/public/templates/scene.html]: I know I wasn't set as a reviewer, but this function isn't being used? Also a console log

Comment (ahmedhamodi) [reply app/public/templates/scene.html]: Oh you read my mind

Comment (jaydhulia) [reply app/public/templates/scene.html]: Already removed it

File: app/public/templates/scene.html [hunk 2]
```diff
@@ -139,7 +139,7 @@
           const next_scene_idx = {{ cur_scene_idx }} + 1;
 
           if({{cur_scene_idx }} == 0){
-            const intro_transition_data = {% raw json.dumps(scenario_transitions[0]) %};
+            const intro_transition_data = {% raw json_encode(scenario_transitions[0]) %};
```
Comment (ahmedhamodi) [inline app/public/templates/scene.html:142]: In the above two cases, you removed the `% raw` but here (and below) it is kept. Any reason?

Comment (jaydhulia) [reply app/public/templates/scene.html:142]: Yeah, those 2 are in HTML context so we need it to be escaped so HTML will read it properly. In these later ones, we are in the "script" tag, so it's interpreted as JS, so need to have it as is (hence raw)

**Review by ahmedhamodi — APPROVED** [review]
lgtm :)

**Review by drbalar — APPROVED** [review]
Looks good 🎉

**Pull request merged**